libs/types/src/lib/websocket.ts

```diff
@@ -79,6 +79,8 @@ export interface WSServerEmittedEvents {
   presentationUpdated: (divisionState: DivisionState) => void;
 
   awardsUpdated: (awards: Array<WithId<Award>>) => void;
+
+  ScheduledTimeChanged: () => void;
```

Comment:
Lets keep our conventions here: `camelCase` and not `PascalCase` for event names.
Also split into 2 events: `matchParticipantTeamUpdated` and `judgingSessionTeamUpdated`. In each event, transmit the session or match that was updated. For matches, also transmit the participant that was updated. This should give us a robust infrastructure we can reuse.

In any case - the schedule time of a match or session will not be changed. Just the team in that slot.